browser/BUILD.gn

```diff
@@ -231,6 +231,8 @@ source_set("browser_process") {
     ]
   } else {
     sources += [
+      "android/brave_cosmetic_resources_tab_helper.cc",
```

Comment:
there are a lot of deps in brave_cosmetic_resources_tab_helper and I don't see anything added here, but we need to separate these out or it's going to be impossible to manage. Can you please create `brave/browser/android/BUILD.gn`? We need to stop creating giant targets like this.

Reply:
I created a follow up https://github.com/brave/brave-browser/issues/11653 to move everything to a separate BUILD.gn file https://github.com/brave/brave-browser/issues/11653